workspaces/client/app/repository/calendar.ts

```diff
@@ -0,0 +1,71 @@
+import { client } from "~/utils/hono";
+
+export interface ICalendarEvent {
+	id: string;
+	userId: string;
+	title: string;
+	description: string;
+	startAt: string;
+	endAt: string;
+}
+
+export interface CreateEventPayload {
```

Comment:
```suggestion
export interface ICreateEventPayload {
```

interface には `I` つけてるので統一しておく？